<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>getter</class>
 <widget class="QWidget" name="getter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Search by city:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="city_line"/>
     </item>
     <item>
      <widget class="QPushButton" name="city_find_button">
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="city_list"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Search by marital status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Married_Radio">
       <property name="text">
        <string>Married</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Single_Radio">
       <property name="text">
        <string>Single</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="marital_find_button">
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="marital_list"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Search by having childrens</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="YesKids_Radio">
       <property name="text">
        <string>Have kids</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="NoKids_Radio">
       <property name="text">
        <string>With no kids</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="childrens_find_button">
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="childrens_list"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
